.NET/Microsoft.Recognizers.Text.DataTypes.DataDrivenTests/TestTimexResolver.cs

```diff
@@ -305,6 +306,42 @@ public void DataTypes_Resolver_DateRange_Last_Three_Weeks()
             Assert.AreEqual("2019-05-01", resolution.Values[0].End);
         }
 
+        [TestMethod]
+        public void DataTypes_Resolver_DateRange_Demaical_Period_PT()
+        {
+            var sourceLanguage = CultureInfo.InstalledUICulture;
```

Comment:
Isn't it enough to only read and set `CurrentCulture` instead of also `InstalledUICulture` and `CurrentUICulture`?